be/src/column/datum.h

```diff
@@ -34,6 +34,7 @@ using DatumArray = std::vector<Datum>;
 using DatumKey = std::variant<int8_t, uint8_t, int16_t, uint16_t, uint24_t, int32_t, uint32_t, int64_t, uint64_t,
                               int96_t, int128_t, Slice, decimal12_t, DecimalV2Value, float, double>;
 using DatumMap = std::map<DatumKey, Datum>;
+using DatumStruct = std::vector<Datum>;
```

Comment:
change DatumArray and DatumStruct to DatumVec

Reply:
I think using two variables is more convenient for others to distinguish different `Datum`.